dev-infrastructure/modules/cluster-service.bicep

```diff
@@ -8,7 +8,7 @@ param clusterServiceManagedIdentityName string
 param clusterServiceManagedIdentityPrincipalId string
 
 @description('The name of the database to create for CS')
-param csDatabaseName string = 'cluster-service'
+param csDatabaseName string = 'clusters-service'
```

Comment:
Doesnt block merge, but if the CS app expects a different name like `cluster-service` this might cause you some head scratching later :)